addon/templates/components/bootstrap-multi-select.hbs

```diff
@@ -1,27 +1,29 @@
-{{#if hasBlock}}
-  {{#if label}}
-    <label for="{{inputId}}" class="control-label">{{label}}</label>
+<div class="form-group bootstrap-power-select-component {{if hasSuccess 'has-success'}} {{if hasWarning 'has-warning'}} {{if hasError 'has-error'}} {{if required 'required'}}">
```

Comment:
Should we set this class name as something different? Such as `bootstrap-power-select-multiple`?